src/main/asciidoc/index.adoc

```diff
@@ -149,6 +149,16 @@ Generally-speaking, we respect the JUnit extension scoping rules, but here are t
 3. Injecting in a `@BeforeEach` with no parent context or previous `@BeforeAll` injection creates a new instance shared with the corresponding test and `AfterEach` method(s).
 4. When no instance exists before running a test method, an instance is created for that test (and only for that test).
 
+==== Configuring of `Vertx` objects
+
+By default the `Vertx` objects get created with `Vertx.vertx()`, using the default settings for `Vertx`. However you have the ability to configure `VertxOptions` to suit your needs.
+A typical use case would be "extending blocking timeout warning for debugging". To configure the `Vertx` object you must:
+
+1. create a json file with the `VertxOptions` in https://vertx.io/docs/apidocs/io/vertx/core/VertxOptions.html#VertxOptions-io.vertx.core.json.JsonObject-[json format] 
+2. create an environment variable `vertx.parameter.filename` pointing to that file
+
+With these conditions met, the `Vertx` object will be created with the configured options
+
```

Comment:
Can you add a short snippet of such a file?

Reply:
I was looking for some example in the Vert.x core documentation, couldn't find one

Reply:
It's a good opportunity to add one here :-)